.gitignore

```diff
@@ -72,5 +72,6 @@ coverage.xml
 # Front-End #
 #############
 node_modules/
+lib/
```

Comment:
If we ignore the `lib` dir the transpiled files won't be published to npm. We can either check `lib` into git or create an `.npmignore` that doesn't list `lib`. See https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package.